superset/assets/src/SqlLab/utils/reduxStateToLocalStorageHelper.js

```diff
@@ -21,9 +21,11 @@ import { LOCALSTORAGE_MAX_QUERY_AGE_MS } from '../constants';
 const PERSISTENT_QUERY_EDITOR_KEYS = new Set([
   'autorun',
   'dbId',
+  'height',
```

Comment:
what does height mean here?

Reply:
this is used by sql editor resize function. if user resize editing area vs results area, this props saves user's setting.

Reply:
Hey btw, i found a bug in master branch: it seems should use `northPercent`, `southPercent` instead of height values.